spec/system/publishers/publishers_can_extend_a_deadline_spec.rb

```diff
@@ -90,7 +90,11 @@
       end
 
       it "sends an event to analytics" do
-        expect(:publisher_vacancy_relisted).to have_been_enqueued_as_analytics_events
+        expect(:publisher_vacancy_relisted).to have_been_enqueued_as_analytics_event(
+          with_data: {
```

Comment:
This looks different from the others - are we sure that this is correct? (That the data is stringified inside the relist-form key)

Reply:
Yes, this comes from the gem.
When the given value for a data key is a hash, [the gem calls `to_json` over it](https://github.com/DFE-Digital/dfe-analytics/blob/main/lib/dfe/analytics/event.rb#L105-L106). 

It seems needed as `data --> value` field in the BigQ table schema is defined as a STRING.